less/index.less

```diff
@@ -53,7 +53,145 @@ table {
     box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
-}
+	html, body {
+		height: 100%;
+		font-size: 62.5%;
+	
+	}
+
+
+	
+
+	//Variable
+	@main-background:url(../steel.jpg);
+	@text-color:black(20, 19, 19, 0.945);
+	@font:'Audiowide', cursive;
+	//Mixins
+	.round-borders{
+		width:7rem;
+		border-radius: 100%;
+		border:1px solid @text-color;
+		margin-right: 1rem;
+	}
+	.border{
+		border:2px solid black;
+	}
+
+	//start of my css
+
+	.container{
+		background-image: @main-background;
+		background-repeat: no-repeat;
+		background-size: cover;
+		margin:auto;
+		max-width: 880px;
+		border: 2px rgba(0, 0, 0, 0.24) solid;
+		font-size: 2rem;
+		font-family: @font;
+		margin: 10px;
+		color: @text-color;
+		h1{
+			font-size: 4rem;
+			padding-bottom: 1rem;
+
+		}
+		h2{
+			font-size:3rem; 
+			padding-bottom: 1rem;
+			.img1{
+				.round-borders;
+			}
+		}
+		
+
+	}
+	nav{
+		display: flex;
+		justify-content: flex-end;
+		a{
+			text-decoration:none;
+			color:white ;
+			font-size: 1.5rem;
+			padding: 10px 2rem;
+			a:hover{
+				text-decoration: underline;
+				color:rgb(41, 37, 37);	
+			}
+		}
+	}
+
+
+	.picbox{
+		display: flex;
+		justify-content: space-evenly;
+		margin:0 1rem;
+		
+		img{
+			height:30%;
+			width:30%;
+			padding:20px 4rem; 
+
+		}
+		div{
+			padding:20px 4rem;
+			h1{
+				font-size: 4rem;
+
+			}
+		}
+	}
+	@media (max-width:500px) {
+		.picbox{
+			flex-direction: column;
+			align-items: center;
+			img{
+				width: 70%;
+				padding: 0;
+			}
+	}
+	}
+
+	.middle-content{
+		display: flex;
+		justify-content:space-evenly;
+		padding: 30px;
+		
+
+		.skills{
+			width: 49%;
+			.border;
+			padding:2rem;
+		}
+		.ilike{
+			width: 49%;
+			.border;
+			padding:2rem;
```

Comment:
For naming this class, you might want to use a `-` to add clarity (as in: `.i-like`, or `i-enjoy` as what your heading text says for this div).